src/generateToken.ts

```diff
@@ -0,0 +1,31 @@
+import { FastifyRequest as Request, FastifyReply as Reply } from 'fastify';
+import knexConfig from '../knexfile';
+import knex from 'knex';
+
+const knexInstance = knex(knexConfig.development);
+
+async function verifyCredentials(id: number, secret: string): Promise<boolean> {
+    const company = await knexInstance('company').where({ id, secret }).first();
+    return !!company;
+}
+export const generateToken = async function generateToken(request: Request, reply: Reply): Promise<void> {
```

Comment:
No necesitas tener 2 veces el nombre de la función. Incluso puedes refactorizarlo haciendo un arrow function: https://bootcamp.uxdesign.cc/arrow-functions-vs-regular-functions-in-javascript-29db7928d696